index.js

```diff
@@ -64,7 +64,13 @@ function fastifyRedis (fastify, options, next) {
     }
   }
 
-  next()
+  if (!redisOptions.lazyConnect) {
+    return client.info((err, _) => {
+      return err ? next(new Error('fastify-redis tried to connect to an invalid redis instance')) : next()
```

Comment:
I would not loose the origin `err` here.
We could set the error cause

Reply:
I wanted to hide the original `err` because in the majority of the cases the error will be 
```
MaxRetriesPerRequestError: Reached the max retries per request limit (which is 1). Refer to "maxRetriesPerRequest" option for details.
```
I dont know if it is clear enough when the real error is that the connection cannot be established. What you think about it?

Reply:
What would be the error in case of the wrong password?
Moreover if the user google that error, the user will find ioredis related issues that is good I think

Reply:
Yeah i think you're right. See commit [a467f07](https://github.com/fastify/fastify-redis/pull/98/commits/a467f07ccc09393f0db473c3213f17af85ee05ba)